code-challenges.js

```diff
@@ -9,67 +9,185 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
-var temp1 = 85
-var temp2 = 350
-var temp3 = 212
+//create custom function with temp as an arguement
+//set boiling point as variable at 212 degrees
+//run an if/else statement that compares arguement to boiling point variable
+//log results on each if/else statement to declare comparison result
 
+//Wanted to have some fun with the arguement name - yes I know it should be named something relative to what it's purpose is....just wanted to start the problem with some humor!
 
 
+var temp1 = 85;
+var temp2 = 350;
+var temp3 = 212;
+
+const compareBoilPoint = (banana) => {
+  let boilPoint = 212;
```

Comment:
very cool! I like that you created a variable to store a value that is used multiple times! This is cool because if this value ever changes, you'll only need to change the value in one place!